sdk/communication/Shared/src/CommunicationRecordedTestSanitizer.cs

```diff
@@ -1,19 +1,27 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System.Collections.Generic;
-using System.Text.RegularExpressions;
 using Azure.Core;
 using Azure.Core.TestFramework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
 
-namespace Azure.Communication.Sms.Tests
+namespace Azure.Communication.Pipeline
 {
-    internal class SmsClientRecordedTestSanitizer : RecordedTestSanitizer
+    internal class CommunicationRecordedTestSanitizer : RecordedTestSanitizer
     {
         private static readonly Regex s_azureResourceRegEx = new Regex(@"[^/]+?(?=(.communication.azure))", RegexOptions.Compiled);
         private static readonly Regex s_identityInRouteRegEx = new Regex(@"(?<=identities/)([^/]+)", RegexOptions.Compiled);
+        internal const string ConnectionStringEnvironmentVariableName = "COMMUNICATION_CONNECTION_STRING";
+        internal const string ToPhoneNumberEnvironmentVariableName = "TO_PHONE_NUMBER";
+        internal const string FromPhoneNumberEnvironmentVariableName = "FROM_PHONE_NUMBER";
+        /// <summary>
+        /// This is a testing/unsigned token required on the sanitized payloads for the playback mode due to format validation on CommunicationUserCredential constructors.
+        /// </summary>
+        internal const string SanitizedChatAuthHeaderValue = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c";
```

Comment:
Let's remove the comment and encapsulate it in the name: `SanitizedUnsignedUserTokenValue`

Reply:
Done, thanks!